### ISSUE TITLE
`vapor new` should create a barebone project without boilerplate code

### ISSUE THREAD

**Issue by alomvar** [qutheory/vapor#215]
Currently every project created with `vapor new` is based on the vapor example project. That results in boilerplate code which is more suitable for demonstration purposes than actual starting ground for new projects. In example the mustache example functionality and dependency likely is going to be removed every time.

The solution would be to introduce the `example` argument for the `vapor` command line application, which creates an new project based on the example. In parallel the `new` argument will be modified to create a barebone project with just the minimal content necessary to get going (requires a new repository as a source).

**Comment by alomvar** [comment]
This issue is merged into #216.
